cypress/integration/creatorOnboardingFlows/creatorSettings.spec.js

```diff
@@ -54,6 +54,20 @@ describe('Creator Settings Page', () => {
     cy.findAllByRole('radio', { name: /members only/i }).check();
     cy.findAllByRole('radio').should('be.checked');
 
+    // should contain a 'I agree to uphold our Code of Conduct' checkbox field and allow selection upon click
+    cy.findByRole('group', {
+      name: /^finally, please agree to the following:/i,
+    }).should('be.visible');
+    cy.findAllByRole('checkbox').first().check();
+    cy.findAllByRole('checkbox').should('be.checked');
```

Comment:
This (and the below test, too) is a bit hacky, but Cypress was picking up the COC and TOS as links rather than as checkboxes. The `name` for both checkboxes appeared to be blank upon the test's failure even though the appropriate `name`s for both checkboxes show when you inspect the elements via the developer tools. 🤔

Reply:
It was probably picking them up as links because there are links to the COC and TOS?